### ISSUE TITLE
Loss not decrease 

### ISSUE THREAD

**Issue by hcnhatnam** [huoyijie/AdvancedEAST#56]
I trained with COCO dataset(about 16000 image(original about 65000) excluded images that do not contain text) and  resize image for 4 size( [256, 384, 512, 640, 736]). Therefor, I have about 16000*4=64000 image for training set. But when training loss was decrease from 0.85->0.7(early stopping) and result of predict is bad. I use batchsize=4.
